File: wireless/devices/instance-post-example-1/instance-post-example-1.java
// Download the **Next-Gen** twilio-java library from:
// twilio.com/docs/libraries/java#installation-nextgen
import com.twilio.sdk.Twilio;
import com.twilio.sdk.resource.preview.wireless.Device;

public class Example {

  // Find your Account Sid and Token at twilio.com/console
  public static final String ACCOUNT_SID = "{{ account_sid }}";
  public static final String AUTH_TOKEN = "{{ auth_token }}";

  public static void main(String[] args) {
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    Device device = Device.update("DEb8eff34b248d066a31c4a953134e183e")
      .setStatus("active")
      .execute();
      
    System.out.println(device.getFriendlyName());
  }
}

File: rest/usage-triggers/instance-post-example-1/instance-post-example-1.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "{{ account_sid }}"; 
$token = "{{ auth_token }}"; 
$client = new Services_Twilio($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$trigger = $client->account->usage_triggers->get("UT33c6aeeba34e48f38d6899ea5b765ad4");
$trigger->update(array(
        "FriendlyName" => "Monthly Maximum Call Usage",
    "CallbackUrl" => "https://www.example.com/monthly-usage-trigger"
    ));
echo $trigger->date_fired;
